UnityProject/Assets/Scripts/Chat/Chat.Process.cs

```diff
@@ -121,75 +124,6 @@ private static string StripTags(string input)
 		return output;
 	}
 
-	private static string ApplyModifiers(string input, ChatModifier modifiers)
-	{
-		string output = input;
-
-		//Clowns say a random number (1-3) HONK!'s after every message
-		if ((modifiers & ChatModifier.Clown) == ChatModifier.Clown)
-		{
-			int intensity = Random.Range(1, 4);
-			for (int i = 0; i < intensity; i++)
-			{
-				if (i == 0)
-				{
-					output = output + " HONK!";
-				}
-				else
-				{
-					output = output + "HONK!";
-				}
-			}
-		}
-
-		//Sneks say extra S's
-		if ((modifiers & ChatModifier.Hiss) == ChatModifier.Hiss)
-		{
-			//Regex - find 1 or more "s"
-			Regex rx = new Regex("s+|S+");
-			output = rx.Replace(output, Hiss);
-		}
-
-		//Stuttering people randomly repeat beginnings of words
-		if ((modifiers & ChatModifier.Stutter) == ChatModifier.Stutter)
-		{
-			//Regex - find word boundary followed by non digit, non special symbol, non end of word letter. Basically find the start of words.
-			Regex rx = new Regex(@"(\b)+([^\d\W])\B");
-			output = rx.Replace(output, Stutter);
-		}
-
-		//Drunk people slur all "s" into "sh", randomly ...hic!... between words and have high % to ...hic!... after a sentance
-		if ((modifiers & ChatModifier.Drunk) == ChatModifier.Drunk)
-		{
-			//Regex - find 1 or more "s"
-			Regex rx = new Regex("s+|S+");
-			output = rx.Replace(output, Slur);
-			//Regex - find 1 or more whitespace
-			rx = new Regex(@"\s+");
-			output = rx.Replace(output, Hic);
-			//50% chance to ...hic!... at end of sentance
-			if (Random.Range(1, 3) == 1)
-			{
-				output = output + " ...hic!...";
-			}
-		}
-		if ((modifiers & ChatModifier.Whisper) == ChatModifier.Whisper)
-		{
-			//If user is in barely conscious state, make text italic
-			//todo: decrease range and modify text somehow
-			//This can be changed later to other status effects
-			output = "<i>"+output+"</i>";
-		}
-		if ((modifiers & ChatModifier.Mute) == ChatModifier.Mute)
-		{
-			//If user is in unconscious state remove text
-			//This can be changed later to other status effects
-			output = "";
-		}
-
-		return output;
-	}
-
```

Comment:
Moved to Chat.cs

Reply:
It was already a partial of Chat why move it?

Reply:
I'll fix this up soon, just want to get all this stuff merged